python/lsst/afw/fits/SConscript

```diff
@@ -1,3 +1,3 @@
 ## -*- python -*-
 #from lsst.sconsUtils import scripts
-#scripts.BasicSConscript.python(['fitsLib'])
+#scripts.BasicSConscript.pybind11(['fits'])
```

Comment:
We have strict standards about commented-out lines. There should always be an explanation of why the code is being left in place, commented out, and what it will take to get rid of it. In this case some of this is boilerplate and probably best left alone until we can take a final cleanup pass and just throw it all out. But aside from the boilerplate I strongly feel a code comment is necessary if the code is to be left there. Otherwise it's just confusing and frustrating for the reader.